server/inspect.go

```diff
@@ -0,0 +1,56 @@
+package server
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/go-zoo/bone"
+)
+
+// ContainerInfo stores information about containers
+type ContainerInfo struct {
+	Pid         int               `json:"pid"`
+	Image       string            `json:"image"`
+	CreatedTime int64             `json:"created_time"`
```

Comment:
can we format it to `time.RFC3339Nano`? cadvisor is gonna read into that and is more human readable for people calling the route

Reply:
We could convert to RFC3339Nano in cadvisor. (Just to get this in for now and then can modify it.)